src/dom/DomEvent.DoubleTap.js

```diff
@@ -9,6 +9,19 @@ var _touchstart = Browser.msPointer ? 'MSPointerDown' : Browser.pointer ? 'point
 var _touchend = Browser.msPointer ? 'MSPointerUp' : Browser.pointer ? 'pointerup' : 'touchend';
 var _pre = '_leaflet_';
 
+function browserFiresNativeDblClick(e) {
+	// See https://github.com/w3c/pointerevents/issues/171
+	// Note however, that Chrome stopped firing native dblclick for
+	// touch since that issue was written.

```

Comment:
Just tested on android.
Chrome 80 and Firefox 68.5 and Firefox Preview 3.2.1 - all fire dblclick.

Checked with this page: https://rbyers.github.io/eventTest.html

Reply:
@johnd0e you're saying the PR isn't necessary anymore? I figured it's been fixed by now. We can close it if that's the case!

Reply:
No, I'm only saying some facts verified on my android phone.
And I have no desktop with touchscreen to make similar tests.

Also, Safari (incl. iOS) is known for issues with dblclick.

As of this PR in general...
It's obvious that Leaflet code was written long time ago, and many parts should be revised.
But I have not enough data to decide whether proposed here changes can makes things better or not.

Reply:
I don't know. I've been meaning to update the Leaflet version I currently use so i'll loop back here when I do

Reply:
> It's obvious that Leaflet code was written long time ago, and many parts should be revised.

In particular, it's time to revisit the `pointerify` branch and make pointer events first-class citizens, instead of all the current hacks and workarounds.

> I don't know. I've been meaning to update the Leaflet version I currently use so i'll loop back here when I do

Please **do not** submit bugs or comment on existing ones if you haven't checked if they apply to the latest release. It's a waste of time for everybody involved.